cypress/integration/instructors_spec.js

```diff
@@ -0,0 +1,13 @@
+describe('instructors', () => {
+  it('/instructors loads', () => {
+    cy.visit('/instructors')
+    cy.get('img').should('have.length', 24)
```

Comment:
Is this testing the number of instructors on the page? I'd like to chat about testing strategy if you're up for it.

Reply:
Yeah, it's just testing for 24 images (like egghead has now). I didn't put any thought into it, just wanted to stub it out.

Reply:
I'd test this `>1` if that was possible. Something like that. 

We've got an ongoing issue of hyper specified tests at egghead so I've got some trauma 😂

Reply:
We can test if "body" exists or whatever. It's based on Chrome APIs, so it's a similar mindset to using the Chrome console to inspect the DOM. I'll change it to >1